conf/modules.config

```diff
@@ -775,6 +775,13 @@ process {
         ]
     }
 
+    withName: MOTUS_PREPLONG {
+        tag = {"${meta.db_name}|${meta.id}"}
```

Comment:
It might bve safer to also use the `ext.prefix` here too

Reply:
Added [here](https://github.com/nf-core/taxprofiler/commit/36e26e09b86f4e94d9d68e305502f16849080dce) and I am resolving this.